<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_ModInst_EP</class>
 <widget class="QWidget" name="f_ModInst_EP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBtnChoix">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Nombre de Branches</string>
   </property>
   <widget class="QRadioButton" name="choix2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="choix3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="choix4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="choix5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="BtnAccept">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
